packs/servicenow/actions/get.py

```diff
@@ -0,0 +1,11 @@
+from lib.actions import BaseAction
+
+
+class GetAction(BaseAction):
+    def run(self, table, query):
+        try:
+            self.client.table = table
+            response = self.client.get(query)
+            return response
+        except e:
```

Comment:
Help me better understand how to return failure in the python runner. Is there a better way to do this other than returning generic exceptions?